test/basic-querying.test.js

```diff
@@ -220,6 +236,20 @@ describe('basic-querying', function() {
       });
     });
 
+    it('should query sorted desc by order integer field even though there' +
+       'is an async model loaded hook', function(done) {
+      User.find({ order: 'order DESC' }, function(err, users) {
+        if (err) return done(err);
+
+        should.exists(users);
+        should.not.exists(err);
```

Comment:
You are already reporting error on line 242 by the callback. You don't need this line anymore...